### PR TITLE
add z to everything

### LINKED ISSUE #255: SE Linux
Update: Please leave this ticket open if it is ok. Still working on it. Can't get sites inside data to show up  in v hosts php 

I need to do more work on this. 

If you encounter a bug and something does not work, make sure you have done the following and check those boxes before submitting an issue - thank you!

I think we need to specify rw for SE Linux somewhere? 

- [x] Pull latest dockers (e.g.: `docker pull cytopia/<used_docker>`) before running `docker-compose up`
- [ ] Specify used docker versions (php, web and database)
- [ ] Attach logs for php, mysql and webserver (found in `log/` directory)
- [ ] Start with debug mode and attach docker-compose output (`.env` setting `DEBUG_COMPOSE_ENTRYPOINT=1`)
- [ ] Never use different mysql|mariadb versions on the same `HOST_PATH_MYSQL_DATADIR` on existing database files. Different mysql|mariadb versions might upgrade/corrupt existing database files. If you have done that already, start with a different path of `HOST_PATH_MYSQL_DATADIR` (to an empty directory) and try again.

Please also specify the following info:

- [x] Which operating system are you at (Linux, OSX or Windows)
- [x] `docker version`
- [x] `docker-compose version`

### PR DESCRIPTION
**Pull request by 9034725985**
fixes #255

I tested on fedora 28
looks good to me
please test this on a non-SE Linux machine
it should not have any effect there

# Allow SE Linux machines to use 

#### Blindly add z, we can optimize later 

This change enables devilbox to work on SE Linux machines such as Fedora 
#255 

#### Tasks done

* [x] Add z to volumes

### CODE REVIEW

**Comment by cytopia** [comment]
@9034725985 what exactly does `z` do?

**Comment by 9034725985** [comment]
From what I understand, the z allows code in one container to do something in a volume owned by another container. Can you please try it on one of your machines (such as a macOS computer maybe)? 

tl;dr The 'z' option tells Docker that the volume content will be shared between containers.  

> Two suffixes :z or :Z can be added to the volume mount. These suffixes tell Docker to relabel file objects on the shared volumes. The 'z' option tells Docker that the volume content will be shared between containers. Docker will label the content with a shared content label. Shared volumes labels allow all containers to read/write content. The 'Z' option tells Docker to label the content with a private unshared label. 
> https://github.com/rhatdan/docker/blob/e6473011583967df4aa5a62f173fb421cae2bb1e/docs/sources/reference/commandline/cli.md
> If you use selinux you can add the z or Z options to modify the selinux label of the host file or directory being mounted into the container. This affects the file or directory on the host machine itself and can have consequences outside of the scope of Docker.
> The z option indicates that the bind mount content is shared among multiple containers. The Z option indicates that the bind mount content is private and unshared. Use extreme caution with these options. Bind-mounting a system directory such as /home or /usr with the Z option renders your host machine inoperable and you may need to relabel the host machine files by hand.
> $ docker run -d \ -it \ --name devtest \ -v "$(pwd)"/target:/app:z \ nginx:latest
> https://docs.docker.com/storage/bind-mounts/#configure-bind-propagation

https://stackoverflow.com/a/35222815

**Comment by cytopia** [comment]
Thanks again for the PR. I reviewed this and would like to make it configurable via `.env` file.

Can you please point this PR to the `mount-options` instead of `master`. I will then merge it into `mount-options` and add the configuration part to it.

`mount-options` is already pushed to remote

**Comment by 9034725985** [comment]
I updated the branch so you can merge

**Comment by cytopia** [comment]
Awesome, will continue here: https://github.com/cytopia/devilbox/pull/297
